### PR TITLE
Adding upgrade job for openstack-ipi

### PR DESCRIPTION
**Pull request by jhou1**
@liangxia @pruan-rht @JianLi-RH @dis016 PTAL
/assign @liangxia

### CODE REVIEW

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.10-upgrade-from-stable-4.9.yaml [hunk 1]
```diff
@@ -42,6 +55,22 @@ tests:
...
+  interval: 24h
+  steps:
+    cluster_profile: openstack
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: minimal
+      UPGRADE_CHECK_RUN_TAGS: '@4.9 and @gcp-ipi and @upgrade-check'
+      UPGRADE_PRE_RUN_TAGS: '@4.9 and @gcp-ipi and @upgrade-prepare'
```
Comment (liangxia) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.10-upgrade-from-stable-4.9.yaml:68]: Test gcp on OpenStack ?

Comment (jhou1) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.10-upgrade-from-stable-4.9.yaml]: missed this... Thank you

**Pull request merged**